sql/core/src/test/scala/org/apache/spark/sql/UDFSuite.scala

```diff
@@ -904,25 +905,43 @@ class UDFSuite extends QueryTest with SharedSparkSession {
 
   test("SPARK-34663: using java.time.Period in UDF") {
     // Regular case
-    val input = Seq(java.time.Period.ofMonths(11)).toDF("p")
+    val input = Seq(java.time.Period.ofMonths(13)).toDF("p")
+      .select($"p", $"p".cast(YearMonthIntervalType(YEAR, YEAR)).as("p_y"),
```

Comment:
Please, simplify `YearMonthIntervalType(YEAR, YEAR)` to `YearMonthIntervalType(YEAR)`

Reply:
DOne